pkgs/development/python-modules/rxpy/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, fetchFromGitHub, buildPythonPackage, nose }:
+
+buildPythonPackage rec {
+  pname = "rxpy";
+  name = "${pname}-${version}";
+  version = "1.6.0";
+
+  src = fetchFromGitHub {
```

Comment:
can't we use pypi?

Reply:
Yes, Though for future is there some preferred order of sources?

Reply:
Just changed it to pypi. There are no tests with that source.